### ISSUE TITLE
wp.utils.WordCounter is not a constructor

### ISSUE THREAD

**Issue by theleemon** [WordPress/gutenberg#5118]
## Issue Overview
Since 2.2 I'm getting the following error when adding or editing a post:

```
Uncaught TypeError: wp.utils.WordCounter is not a constructor
    at load-scripts.php?c=0&load[]=hoverIntent,common,admin-bar,svg-painter,heartbeat,wp-auth-check,wp-ajax-response,suggest,jquery-color,wp-lists,jquery-ui-positi&load[]=on,jquery-ui-menu,wp-a11y,jquery-ui-autocomplete,tags-suggest,tags-box,post,media-editor,media-audiovideo,mce-view,imgareaselect&load[]=,image-edit,jquery-ui-draggable,jquery-ui-slider,jquery-touch-punch,iris,wp-color-picker,quicktags,wplink,thickbox,media-upload,&load[]=sack,wp-pointer&ver=4.9.4:290
(anonymous) @ load-scripts.php?c=0&load[]=hoverIntent,common,admin-bar,svg-painter,heartbeat,wp-auth-check,wp-ajax-response,suggest,jquery-color,wp-lists,jquery-ui-positi&load[]=on,jquery-ui-menu,wp-a11y,jquery-ui-autocomplete,tags-suggest,tags-box,post,media-editor,media-audiovideo,mce-view,imgareaselect&load[]=,image-edit,jquery-ui-draggable,jquery-ui-slider,jquery-touch-punch,iris,wp-color-picker,quicktags,wplink,thickbox,media-upload,&load[]=sack,wp-pointer&ver=4.9.4:290

```

## Steps to Reproduce (for bugs)

1. Add or edit a post
2. Open browser console
3.
4.

Firefox and Chrome

**Comment by Soean** [comment]
Do you have any other plugins active? If yes, can you test it without the other and activate them step by step?

**Comment by theleemon** [comment]
Ok, I investigated the issue and it seems that the culprit is the Polylang plugin:

https://es.wordpress.org/plugins/polylang/

**Comment by manooweb** [comment]
The culprit is not exactly Polylang. wp.utils.WordCounter is a function of WordPress Core and I reproduce this case only when Gutenberg is activated, Polylang declare one language and when javascript files are concatenated. When I define the constante CONCATENATE_SCRIPTS to false, I have no js error.
I am looking for the cause of this issue.

**Comment by theleemon** [comment]
This is fixed in the latest version of Polylang (2.3.4):

https://github.com/polylang/polylang/issues/225
